catalog/spatial/registry/registry-rest-endpoint/src/main/resources/templates/organizations.hbt

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta content="text/html; charset=windows-1252" http-equiv="content-type"/>
+    <!-- styles -->
+    <link href="/services/registries/assets/css/bootstrap.min.css" rel="stylesheet"/>
+    <style type="text/css">
+       body {
+            margin-left: 15px;
+        }
+        .scrollable-list {
```

Comment:
removed most of the css, took out the h3 and h4 css as well in the summary.hbt since they were not used.